src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -192,27 +197,39 @@ public async Task<Manifests> DeserializeExistingManifestsAndUpdate(List<string>
         /// <returns>Boolean representing whether the manifest was updated successfully or not.</returns>
         public async Task<bool> ExecuteManifestUpdate(List<string> latestManifestContent, CommandExecutedEvent commandEvent)
         {
-            Manifests manifests = await this.DeserializeExistingManifestsAndUpdate(latestManifestContent);
-            if (manifests == null)
+            Manifests originalManifests = new Manifests();
+            Serialization.DeserializeManifestContents(latestManifestContent, originalManifests);
+
+            DisplayManifestPreview(originalManifests);
+
+            Manifests updatedManifests = await this.UpdateExistingManifests(originalManifests);
+            if (updatedManifests == null)
             {
                 return false;
             }
 
-            DisplayManifestPreview(manifests);
+            DisplayManifestPreview(updatedManifests);
 
             if (string.IsNullOrEmpty(this.OutputDir))
             {
                 this.OutputDir = Directory.GetCurrentDirectory();
             }
 
-            string manifestDirectoryPath = SaveManifestDirToLocalPath(manifests, this.OutputDir);
+            string manifestDirectoryPath = SaveManifestDirToLocalPath(updatedManifests, this.OutputDir);
 
             if (ValidateManifest(manifestDirectoryPath))
             {
                 if (this.SubmitToGitHub)
                 {
+                    if (!this.CheckForUpdatedInstallerHash(originalManifests, updatedManifests))

```

Comment:
>CheckForUpdatedInstallerHash [](http://example.com/codeflow?start=30&length=28)

I'd change name to something like VerifyUpdated...

Reply:
Changed to VerifyUpdatedInstallerHash